language/en/phpbb_ideas_acp.php

```diff
@@ -23,8 +23,9 @@
 	'ACP_IDEAS_FORUM_ID'			=> 'Ideas forum',
 	'ACP_IDEAS_FORUM_ID_EXPLAIN'	=> 'Select the forum that Ideas topics will be posted to. Once done, apply Ideas forum setup clicking the appropriate button below.',
 	'ACP_IDEAS_FORUM_SETUP'			=> 'Set up the Ideas forum',
+	'ACP_IDEAS_FORUM_SETUP_CONFIRM'	=> 'Are you sure you wish to set up the Ideas forum?',
```

Comment:
This is just consistent to how is it worded in the core, f.e. https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/language/en/acp/common.php#L381